platform/android/app/build.gradle

```diff
@@ -257,3 +258,10 @@ tasks.whenTaskAdded { task ->
     )
         task.dependsOn copyTestFeatureOntoDevice
 }
+
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
+    kotlinOptions {
+        // Generated FlatBuffer models use ExperimentalUnsignedTypes
+        freeCompilerArgs += "-opt-in=kotlin.ExperimentalUnsignedTypes"
```

Comment:
I find it non-trivial to check what Kotlin version we're on, but it appears that if we are on >=1.5, and potentially also a newer version of Flatbuffers, we won't need to do this, see: https://github.com/google/flatbuffers/issues/7087

Reply:
That's correct, but latest FlattBuffers release with precompiled `flatc` is `2.0.0`. And we want to avoid compiling `flatc` locally :)